sklearnex/covariance/__init__.py

```diff
@@ -0,0 +1,19 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+from .incremental_covariance import IncrementalEmpiricalCovariance
+
+__all__ = ["IncrementalEmpiricalCovariance"]
```

Comment:
So we will have online covariance out of preview but standard covariance in preview?

Reply:
@olegkkruglov Please leave here some description why you are making this as resolved

Reply:
i have answered in private channel.

Reply:
It's always better to make it as clear as possible why the discussion is over.
It could be very useful for other reviewers